docs/source/plotting/index.rst

```diff
@@ -472,6 +468,45 @@ Both can be found `online <https://www.rapidtables.com/web/color/html-color-code
    fig.suptitle('Line Plots: Color Cycle', fontsize='x-large')
    #fig.show()
 
+Custom Colormap (MantidPlot)
```

Comment:
This example should be removed as it references MantidPlot (see [this pr](https://github.com/mantidproject/mantid/pull/31466/files#diff-baae6b8119087370c6e61703bfd8615dee363ddf9b32de628c1bcc8298a8c709)). I think you may have started this branch before release-next merged with master so it snuck back in.